OTEP.md

```diff
@@ -194,7 +194,7 @@ otel.ParseAndValidateConfiguration(config{
 
 ### Configuration file
 
-The configuration model *MUST* also be configurable via the use of a configuration file. The working group proposes that all implementations *MUST* support JSON as a configuration file format, and *SHOULD* support YAML.
+The configuration model *MUST* also be configurable via the use of a configuration file. The working group proposes that all implementations *SHOULD* support YAML as a configuration file format, and *MAY* support JSON.
```

Comment:
Doesn't feel quite right that we have to use this normative language about file formats in multiple places (it also occurs up on line 56. Maybe we just delete this line? Don't feel strongly though.

Reply:
I'm ok with removing it altogether here.